aemedge/blocks/content-list/content-list.js

```diff
@@ -18,21 +23,14 @@ function matchAuthors(entry, config) {
   return config.authors.some((item) => authors.includes(item.trim()));
 }
 
-function matchTopics(entry, config) {
```

Comment:
Does removing this mean we are not filtering by topics any more?

Reply:
Yes because now we only have tags in index. I still have kept method to filter by content type by extracting it from index entry. We can do the same for topic as well. Let me reinstate the method.

Reply:
I actually meant filter by tags. Sorry for being unclear here. Topics as one group of tags.

We need filter by tag or tag group. For example:

- filter by topic/ai
- filter by industry/retail
Etc. 

But also:

- filter by topic
- filter by industry
Which will show all tagged with any topic or any industry. 

Topic and industry are only samples here we also need support for all other tags.

Reply:
In that case lets do a n level filtering as well. So 

- filter by topics/ai/responsible-ai filters all articles specifically tagged with "/topics/ai/responsible-ai"
- filter by topics/ai filters all articles tagged with 'topic/ai, topics/ai/responsible-ai, topics/ai/generative-ai'

Same goes for industry, news-topic, product etc. So we don't have to make code changes again and they can play with tagging hierarchy.

Reply:
Yes this it is aka. should be it like :-)

Reply:
fixed